apps/thehive/analyzers/VAST/Dockerfile

```diff
@@ -0,0 +1,50 @@
+ARG VAST_VERSION=latest
+ARG PYTHON_VERSION=latest
+
+FROM tenzir/vast:$VAST_VERSION as vast
+
+FROM python:$PYTHON_VERSION
```

Comment:
Valid point. I was thinking to give users ability to choose python version (if they would like to update or try things on Analyzer/Responder) but it seems it may be better to just use VAST base image. I'll check Python version, applicability and comment here with a commit or reasoning.